cypress/support/step_definitions/dashboard/listitems/the_textarea_should_show_if_i_click_the_request_changes_radio_button.js

```diff
@@ -1,5 +0,0 @@
-And("The textarea should show if I click the Request changes radio button", () => {
```

Comment:
This is more of a "then" but I think it was used as a "when".


Given - set up the state before a test
When - usually an action, or something changing etc
Then - the thing you want to validate. 

Typically you would not see "if" in smoke test definition either. Maybe in the feature or scenario name, but "if" isn't deterministic. in GIVEN WHEN THEN - it would look more like "... when I click the request changes button THEN I see the text area".